framework/molecules/timelines/timelines.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  title: 'Timelines',
+  label: 'Timelines',
+  status: 'wip',
+  collated: true,
+  preview: '@preview-center-transparent',
+  context: {
+    _demo: {
+      scripts: 'Europa.timeline();',
```

Comment:
It's just for the demo, to show how to include a JS function call in a pen. I agree it does nothing interesting and is not mandatory, but it also reminds us that the JS behaviour is not implemented yet